docs/react-native-testing-library/example-intro.mdx

```diff
@@ -32,20 +32,19 @@ function Example() {
 }
 
 test('examples of some things', async () => {
-  const {getByTestId, getByText, queryByTestId, toJSON} = render(<Example />)
-  const famousProgrammerInHistory = 'Ada Lovelace'
-
-  const input = getByTestId('input')
-  fireEvent.changeText(input, famousProgrammerInHistory)
-
-  const button = getByText('Print Username')
-  fireEvent.press(button)
-
-  await waitFor(() => expect(queryByTestId('printed-username')).toBeTruthy())
-
-  expect(getByTestId('printed-username').props.children).toBe(
-    famousProgrammerInHistory,
-  )
+  const expectedUsername = 'Ada Lovelace'
+  
+  render(<Example />)
+
+  fireEvent.changeText(screen.getByTestId('input'), expectedUsername)
+  fireEvent.press(screen.getByText('Print Username'))
```

Comment:
Could we use role matchers here?
```
screen.getByRole('input');
```

Reply:
I've found a curious thing that RN has neither `input` nor `textbox` accessibility role. In fact there is suitable default text input role.